### PR TITLE
Added the release notes with new version 1.16.1.0 in new format

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Added the release notes with new version 1.16.1.0 in the new format.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [X] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/09_11610_rel_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Version 1.16.1.0"
+---
+
+New features, enhancements, bug fixes, and other changes in Pgbouncer 1.16.1.0 include the following:
```
Comment (drothery-edb) [inline product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/09_11610_rel_notes.mdx:5]:
```suggestion
New features, enhancements, bug fixes, and other changes in PgBouncer 1.16.1.0 include the following:
```

File: product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/09_11610_rel_notes.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+| ------------ |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Security Fix | Make PgBouncer acting as a server reject extraneous data after an SSL or GSS encryption handshake.<br />A man-in-the-middle with the ability to inject data into the TCP connection could stuff some cleartext data into the start of a supposedly encryption-protected database session. This could be abused to send faked SQL commands to the server, although that would only work if PgBouncer did not demand any authentication data. However, a PgBouncer setup relying on SSL certificate authentication might well not do so.|
```
Comment (drothery-edb) [inline product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/09_11610_rel_notes.mdx:9]: Typically we want to use past tense for describing bug fixes. 
Is "stuff" the appropriate technical term?
Add commas around the phrase "acting as a server"

File: product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/10_11601_rel_notes.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Version 1.16.0.1"
+---
+
+New features, enhancements, bug fixes, and other changes in Pgbouncer 1.16.0.1 include the following:
```
Comment (drothery-edb) [inline product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/10_11601_rel_notes.mdx:5]:
```suggestion
New features, enhancements, bug fixes, and other changes in PgBouncer 1.16.0.1 include the following:
```

**Review by drothery-edb — APPROVED** [review]
If you have the time, it would be nice to add the Upstream Merge row to the tables like I did in the pgpool rel notes. Maybe change the intro sentence like I did in the pgpool rel notes too.

**Review by drothery-edb — APPROVED** [review]

File: product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/09_11610_rel_notes.mdx (hunk 2 above)
Comment (ebgitelman) [reply product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/09_11610_rel_notes.mdx]: Remove the word "well".in the last sentence.

**Pull request merged**